src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -342,6 +342,13 @@ void AddSection(string kind, ImmutableArray<TaggedText> taggedParts)
             }
             else if (documentedSymbol is object)
             {
+                if (documentedSymbol.IsInSystemNamespace())
+                {
+                    // Don't show <remarks> content for items in the System namespace (or any child namespace).
+                    // https://github.com/dotnet/roslyn/issues/42606
+                    return default;
+                }
```

Comment:
weren't there cases of types in UIKit that had very large remarks.  I'm curious how many of those there are.  Could we ask teh mono people for some data here?

Reply:
Agreed we should really be getting more data here before excluding specific namespaces.

Reply:
I think it's a lot in UIKit and other Xamarin namespaces. @davidortinau is probably the best person to weigh in here from the Xamarin side. He's out today but I'll make sure he's aware of the situation.